### PR TITLE
Create endpoint to accept Prebid analytics

### PR DESCRIPTION
**Pull request by kelvin-chappell**
This is a very simple endpoint to log analytics from a small sample of Prebid auctions.
It will be built on to get this data into the data lake.

See https://github.com/guardian/Prebid.js/pull/20

### CODE REVIEW

File: commercial/app/controllers/PrebidAnalyticsController.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import model.Cached.WithoutRevalidationResult
+import model.{CacheTime, Cached, TinyResponse}
+import play.api.libs.json.JsValue
+import play.api.libs.json.Json.prettyPrint
+import play.api.mvc._
+
+class PrebidAnalyticsController(val controllerComponents: ControllerComponents) extends BaseController with Logging {
+
+  private def serve404[A](implicit request: Request[A]) =
+    Cached(CacheTime.NotFound)(WithoutRevalidationResult(NotFound))
```
Comment (JonNorman) [inline commercial/app/controllers/PrebidAnalyticsController.scala]: Why is this using a `WithoutRevalidationResult` and not a `RevalidatableResult`?

File: commercial/app/controllers/PrebidAnalyticsController.scala [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+  def insert(): Action[JsValue] = Action(parse.json) { implicit request =>
+    if (prebidAnalytics.isSwitchedOn) {
+      log.info(prettyPrint(request.body))
```
Comment (JonNorman) [inline commercial/app/controllers/PrebidAnalyticsController.scala]: So are we going to collect our usage stats through the ELK stack?

Comment (kelvin-chappell) [reply commercial/app/controllers/PrebidAnalyticsController.scala]: Initially.  This is stage 1 in a 7 stage process.  To prove the analytics gathering process with a small sample.

File: commercial/app/controllers/PrebidAnalyticsController.scala (hunk 1 above)
Comment (kelvin-chappell) [reply commercial/app/controllers/PrebidAnalyticsController.scala]: I don't really know what it means.  I just pinched it.  I didn't want it to be revalidated if the switch was off.  So it's meant to be pessimistic.

**Pull request merged**